src/agent/onefuzz-task/src/tasks/report/dotnet/common.rs

```diff
@@ -208,7 +212,8 @@ pub fn parse_sos_print_exception_output(text: &str) -> Result<DotnetExceptionInf
         }
     }
 
-    let exception = exception.ok_or_else(|| format_err!("missing exception type"))?;
+    let exception =
+        exception.ok_or_else(|| format_err!("missing exception type, output was:\n{}", text))?;
```

Comment:
really we should collect up all the missing info and report it all at once (is there a `Validation` type?)